packages/internal/src/generate/templates/all-currentUser.d.ts.template

```diff
@@ -6,9 +6,20 @@ import { getCurrentUser } from '../../../api/src/lib/auth'
 type ThenArg<T> = T extends PromiseLike<infer U> ? U : T
 export type InferredCurrentUser = ThenArg<ReturnType<typeof getCurrentUser>>
 
+type UndefinedRoles = {
+  /**
+   * ⚠️ Heads-Up: This is undefined unless roles key is returned from {@link getCurrentUser()}
+   *
+   * You may look into to https://redwoodjs.com/docs/tutorial/chapter7/rbac for a tutorial on
```

Comment:
```suggestion
   * You may look into https://redwoodjs.com/docs/tutorial/chapter7/rbac for a tutorial on
```

Reply:
Good catch :laughing: – discarding this in favor of your alternative suggestion below.